lib/proofer/vendor/mock.rb

```diff
@@ -109,6 +112,14 @@ def build_question_set(_vendor_resp)
 
       private
 
+      def financial_errors(financials)
+        errors = {}
+        financials.keys.each do |key|
+          errors[key] = "The #{key} could not be verified."
+        end
+        errors
```

Comment:
This is not a request for change, just a Ruby pro tip that whenever you see this pattern where you initialize a hash or array, then iterate through something to populate the hash or array, then return the final hash or array, you can use `each_with_object` instead:
```ruby
financials.keys.each_with_object({}) do |key, result|
  result[key] = "The #{key} could not be verified."
end
```